front/main/app/mixins/discussion-moderation-model.js

```
@@ -128,6 +129,60 @@ export default Mixin.create({
 	},
 
 	/**
+	 * Sets up reported details on reported posts
+	 * @param {DiscussionEntities} entities
+	 * @returns {void}
+	 */
+	reportedDetailsSetUp(entities) {
+		const reportedEntities = entities.filterBy('isReported', true);
+
+		if (!reportedEntities.length) {
+			return;
+		}
+
+		const entitiesReportDetails = {};
+
+		request(M.getDiscussionServiceUrl(`/${this.wikiId}/reports`), {
+			data: {postId: reportedEntities.mapBy('id')},
+			method: 'GET',
+			traditional: true,
+		}).then((data) => {
+			Ember.get(data, 'posts').forEach((reportDetailsData) => {
+				entitiesReportDetails[reportDetailsData.postId] = ReportDetails.create(reportDetailsData);
+			}, this);
+
+			reportedEntities.forEach((reportedEntity) => {
+				reportedEntity.set('reportDetails', entitiesReportDetails[reportedEntity.get('id')]);
+			});
+		}).catch();
```

Could you make sure this empty catch() block will catch any rejected promises / throws? :)